### DISCUSSION TITLE
Parsing Array with `getQuery`

### DISCUSSION THREAD

**Discussion by peterkimzz** [nuxt/nuxt#27181] [Questions]
Hi,

When I send a request with an array of querystring value with axios like below, seems a little bit weird.

```ts
axios.get('nuxt-serevr-url', { params: { ids: ["1"] } })

// nuxt server
const query = await getQuery<{ "ids[]"?: string | string[] }>(event);
console.log(query)
// { "ids[]": "1" } -> string
```

```ts
axios.get('nuxt-serevr-url', { params: { ids: ["1", "2"] } })

// nuxt server
const query = await getQuery<{ "ids[]"?: string | string[] }>(event);
console.log(query)
// { "ids[]": ["1", "2"] } -> string[]
```

Is there any better solutions to parse array querystring value in nuxt3?

**Comment by xjccc** [comment]
You could look at `ufo` for this behavior.

[withQuery](https://github.com/unjs/ufo/blob/666091db37a03dc871e25d3621aeff1b89648196/test/query.test.ts#L45)
[getQuery](https://github.com/unjs/ufo/blob/666091db37a03dc871e25d3621aeff1b89648196/test/query.test.ts#L82)
